src/style.css

```diff
@@ -1,3 +1,75 @@
 body {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
   margin: 0;
 }
+
+html {
+  font-family: "Courier New", Courier, monospace;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.logo__image {
+  height: 100%;
+  width: 100%;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
```

Comment:
![2023-10-30_15h12_03](https://github.com/mate-academy/layout_moyo-header/assets/61904995/d61223d2-7002-4bac-87eb-343e27e40c98)
no fixed
```suggestion
  padding: 23px 0;
```